resources/views/site/realops.blade.php

```diff
@@ -0,0 +1,124 @@
+@extends('layouts.master')
+
+@section('title')
+Realops
+@endsection
+
+@section('content')
+<span class="border border-light" style="background-color:#F0F0F0">
+    <div class="container py-4">
+        @if(auth()->guard('realops')->guest())
+            <a href="/realops/login" class="btn btn-primary float-right">Login as Realops Pilot</a>
+        @else
+            <button disabled class="btn btn-primary float-right">Welcome, {{ auth()->guard('realops')->user()->full_name }}</button>
+        @endif
+        <h2>Realops</h2>
+    </div>
+</span>
+<br>
+
+<div class="container">
+
+<p>Welcome to the main page for ZTL's Realops!</p>
```

Comment:
I feel like we should expand this to further explain what is going on here. I saw that you asked Adam for additional description text. I would recommend something like: "Welcome to the main page for ZTL's Realops event! Realops events simulate actual traffic flow by encouraging pilot to fly real-world routes flown by actual airlines on a real time schedule. Use the controls below to bid on a flight and manage your event participation. Thanks for flying with ZTL!"

Reply:
Ah good idea - I never got a response from Adam about that. I'll run the wording by Eli to get his opinion.

Reply:
Resolved and squashed into 94cd222